en/test_and_ci/docker.md

```diff
@@ -126,13 +129,17 @@ xhost +
 # Run docker and open bash shell
 docker run -it --privileged \
 --env=LOCAL_USER_ID="$(id -u)" \
--v ~/src/PX4-Autopilot:/src/PX4-Autopilot/:rw \
+-v ~/Projects/PX4-Autopilot:/src/PX4-Autopilot/:rw \
 -v /tmp/.X11-unix:/tmp/.X11-unix:ro \
--e DISPLAY=:0 \
--p 14570:14570/udp \
---name=mycontainer px4io/px4-dev-ros:2017-10-23 bash
+-e DISPLAY=$DISPLAY \
+--network host \
+--name=px4-ros px4io/px4-dev-ros2-foxy:2022-07-31 bash
 ```
 
+:::note
+We use the host network mode to avoid conflicts between the UDP port access control when using QGroundControl on the same system as the docker container.
+:::
```

Comment:
THanks @magicbycalvin 

@julianoes @JonasVautherin Do either of you know enough to evaluate this use of `--network host` change? Or can you recommend someone?

Reply:
I'm never using this script, so I can't really comment on it. I also don't know what port forwarding 14570 was for.

Reply:
@julianoes Can you recommend who might know/who else I could ask?

Reply:
I personally don't use my containers with `--network host`, but I don't really see a big problem in recommending it here. It's a pretty basic Docker feature, so people who know Docker will be fine removing it if they want to.

Overall if this helps beginners getting started, why not :+1:.

Reply:
As long as it works. Thanks!